doc/en/components/grids/_shared/column-resizing.md

```diff
@@ -109,6 +112,19 @@ public onResize(event) {
 }
 ```
 
+```tsx
+function onResize(grid: IgrGridBaseDirective, event: IgrColumnMovingEventArgs) {
+  IgrColumn col = event.detail.column;
+  string pWidth = event.detail.prevWidth;
+  string nWidth = event.detail.newWidth;
+}
+
+<{ComponentSelector} id="grid" autoGenerate="false" columnResized={onResize}>
```

Comment:
@mddragnev , this resolves to `igr-grid` . We need to fix the placeholder value if we are to use it.

Reply:
There is a pending PR fixing this issue: https://github.com/IgniteUI/igniteui-xplat-docs/pull/949